spec/pagseguro/features/create_transaction_request_spec.rb

```diff
@@ -37,8 +37,8 @@
         status: [400, "Bad Request"], body: body, content_type: "text/xml"
     end
 
-    it "returns false" do
-      expect(transaction.create).to be_falsey
+    it "returns transaction" do
+      expect(transaction.create).to eq(transaction)
```

Comment:
@lucasrenan ok :). I thought the project will change all to `register` and return a instance :P .

Not merge I'll update this PR :)